io.github.yuxshao.ptcollab.yml

```
@@ -19,5 +19,8 @@ modules:
       - type: git
         url: https://github.com/yuxshao/ptcollab.git
         tag: "v0.6.4.8"
-      - type: patch
-        path: metainfo.patch
+      - type: file
+        path: io.github.yuxshao.ptcollab.metainfo.xml
+    post-install:
+      - mkdir -p/app/share/metadata/
+      - cp -v io.github.yuxshao.ptcollab.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```

```suggestion
      - install -Dm644 io.github.yuxshao.ptcollab.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
```

I already suggested this in the past and was ignored. But your commands won't work either. A local build would have told you so.